flocker/ca/_script.py

```diff
@@ -16,10 +16,74 @@
 from ..common.script import (flocker_standard_options, ICommandLineScript,
                              FlockerScriptRunner)
 
-from ._ca import (CertificateAuthority, CertificateAlreadyExistsError,
-                  KeyAlreadyExistsError)
+from ._ca import (RootCredential, ControlCredential,
+                  CertificateAlreadyExistsError, KeyAlreadyExistsError,
+                  PathError)
 
 
+@flocker_standard_options
```

Comment:
OK, so, both commands have serious output problems, with our without standard options. Including standard options does result in better output, but:

1. Linebreaks aren't preserved, so output is wonky. I think this is a Twisted "feature", not clear if there's any work to around it but at least we don't to look terrible like we do now.
2. `flocker-ca initialize --help` (and same for this command) has different output than `flocker-ca initialize`; in particular notice `<name>` is missing from the latter.
3. `flocker-ca --help` and `flocker-ca` should have usage `flocker-ca <command> [command-options]`.